docs/modules/vanish.md

```diff
@@ -8,4 +8,16 @@ modulename: Vanish
 
 ## Introduction
 
-The Vanish module allows players to turn themselves invisible to other entities on the server.
\ No newline at end of file
+The Vanish module allows players to turn themselves invisible to other entities on the server. This is a good way to hide yourself while doing work on your server in peace when other players are on your server. 
+
+## Configuration
+
+Don't be deceived when looking at the configuration options in your `main.conf`file. While only a couple options reside in the vanish section of your configuration file, other modules have options for `/vanish`. These modules are:
```

Comment:
This is a very good point, but you should also mention the configuration options available for the `vanish` module.